### ISSUE TITLE
When bluetooth headphone connects, this addon does not detect connection unless I restart nvda

### ISSUE THREAD

**Issue by hamidrezaabroshan** [mltony/nvda-bluetooth-audio#7]
When I connect headphone, this addon should be notified and start playing audio. but it does not work before nvda gets restart.
Thanks
